apps/explorer/src/app/components/txs/id/tx-content.tsx

```diff
@@ -1,36 +1,39 @@
 import { ChainExplorerTxResponse } from '../../../routes/types/chain-explorer-response';
-import { Table } from '../../table';
 import { SyntaxHighlighter } from '../../syntax-highlighter';
+import { Table, TableRow, TableHeader, TableCell } from '../../table';
+import { TxOrderType } from '../tx-order-type';
+import { StatusMessage } from '../../status-message';
 
 interface TxContentProps {
   data: ChainExplorerTxResponse | undefined;
 }
 
 export const TxContent = ({ data }: TxContentProps) => {
   if (!data?.Command) {
-    return <>Awaiting decoded transaction data</>;
+    return (
+      <StatusMessage>Could not retrieve transaction content</StatusMessage>
+    );
   }
 
-  const { marketId, type, side, size } = JSON.parse(data.Command);
-
-  const displayCode = {
-    market: marketId,
-    type,
-    side,
-    size,
-  };
-
   return (
     <>
```

Comment:
Unessecary fragment

Reply:
Needed a parent fragment, but have changed the structure a little as I don't need the conditional rendering that follows the table.